unity/Assets/Scripts/CollisionListener.cs

```diff
@@ -31,8 +31,20 @@ public class StaticCollision
             }
         }
     }
+
     private HashSet<Collider> activeColliders = new HashSet<Collider>();
+    public static bool useMassThreshold = false;
+    public static float massThreshold = 10f;
```

Comment:
Why is this static, but you set the value using an instance of collisionListener? If you want the value to apply to all the CollisionListeners, you could just change the setter to public static get/set property or set the value directly on CollisionListener (class not instance).

Reply:
Ah yes I see, I think we do want this to apply to all CollisionListeners in case we ever have multiple agents, which would then have multiple instances of the CollisionListener component existing in the same scene.

I will change this to set the values directly via the CollisionListener class.